src/core/server/saved_objects/migrationsv2/actions/integration_tests/actions.test.ts

```diff
@@ -818,8 +817,8 @@ describe('migration actions', () => {
         _tag: 'Left',
         left: {
           error: expect.any(ResponseError),
-          message: expect.stringMatching(
-            /\[timeout_exception\] Timed out waiting for completion of \[org.elasticsearch.index.reindex.BulkByScrollTask/
+          message: expect.stringContaining(
+            '[timeout_exception] Timed out waiting for completion of',
```

Comment:
Switched to a substring matching both the `v7.x` and `v8.x` version of the response returned by ES